src/SimpleCalculator.Web/Controllers/HomeController.cs

```diff
@@ -1,23 +1,28 @@
 ﻿using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
 using SimpleCalculator.Web.Models;
+using SimpleCalculator.DataAccessLibrary.Data;
+using SimpleCalculator.DataAccessLibrary.Model;
 using Calculations;
 
 namespace SimpleCalculator.Web.Controllers;
 
 public class HomeController : Controller
 {
+    private readonly CalculatorDbContext _context;
     private readonly IAddition _addition;
     private readonly ISubtraction _subtraction;
     private readonly IMultiplication _multiplication;
     private readonly IDivision _division;
 
-    public HomeController(IAddition addition, ISubtraction subtraction, IMultiplication multiplication, IDivision division)
+    public HomeController(IAddition addition, ISubtraction subtraction,
```

Comment:
nitpick: controller become bigger, it's time to create a service and move a business logic there, additionally you can get rid of duplicate logic of saving results to db